src/app/style/abstract/_media.scss

```diff
@@ -10,25 +10,46 @@
  */
 
 @mixin desktop {
-    @media (min-width: 1025px) {
+    @media (min-width: 768px) {
         @content;
     }
 }
 
+// should be removed in future releases
+// kept as legacy support
 @mixin before-desktop {
-    @media (max-width: 1024px) {
+    @include mobile {
         @content;
     }
 }
 
 @mixin tablet {
-    @media (max-width: 1024px) and (min-width: 768px) {
+    @media (min-width: 768px)
```

Comment:
Why has this change been made?

Reply:
To avoid media query overlap max-width should 1 px smaller than target size.